packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -190,12 +195,13 @@ export const Dropdown = forwardRef(
             })
         ) : (
             <MoreIcon
-                isToggled={isToggled}
-                size={24}
+                size="small"
+                variant="tertiary"
                 icon="MORE"
-                color={!isDisabled ? theme.TYPE_DARK_GREY : theme.TYPE_LIGHT_GREY}
-                $isDisabled={isDisabled}
-                {...rest}
+                tabIndex={-1}
```

Comment:
Are you sure about this? It will not be available with TAB, am I right?

Reply:
Since `Dropdown` allows for a custom component to act as the anchor for the dropdown, the keyboard navigation logic is not covered by this `IconButton` but rather the `Container`. `IconButton`, being a `button`, was actually stealing the focus from `Container`, causing the bug you have pointed out in your video from the secon point in your first comment :)